index.go

```diff
@@ -56,6 +60,41 @@ func (idx *Index) Invisible() *Index {
 	return &tmp
 }
 
+// ASC returns a copy of idx with the ASC option.
+// If you set both ASC and DESC, the last one will be used.
+// If you specify the non-existent column, it will be ignored.
+func (idx *Index) ASC(column string) *Index {
+	tmp := *idx // shallow copy
```

Comment:
please create a copy of `idx.order`.

Reply:
Fix [cc33cf2](https://github.com/shogo82148/myddlmaker/pull/78/commits/cc33cf27e4866ca57c48ff14f52c0b9bac510156)